cocotb/share/lib/gpi/GpiCommon.cpp

```diff
@@ -33,6 +33,7 @@
 #include <unistd.h>
 #include <vector>
 #include <map>
+#include <algorithm>
```

Comment:
```suggestion
#include <algorithm>        // std::find
#include <string>           // std::string
```

I think we should start adding comments next to includes to justify their inclusion. Also IWYU,.

Reply:
I'd argue that `IWYU` would make the comments redundant, but I suppose they're marginally useful until then.

Good catch on including `std::string`.

Reply:
@themperek, you should take either @ktbarrett's change above, or the slightly smaller
```suggestion
#include <algorithm>
#include <string>
```